runtime/rasdump/javadump.cpp

```diff
@@ -1770,12 +1770,55 @@ JavaCoreDumpWriter::writeMemoryCountersSection(void)
 			writeNativeAllocator("Other", i + 1, 0, frame->liveBytes, frame->liveAllocations);
 		}
 	}
+        /* Write the section trailer */
+        _OutputStream.writeCharacters(
+                "NULL\n"
+                "NULL           ------------------------------------------------------------------------\n"
+        );
+
+#ifdef J9ZTPF
+	if (mmap(NULL, 0, 0, MAP_SUPPORTED, 0, 0) != MAP_FAILED)  {
+		rc = mprivatestats(&ztpfNativeStats);
+
+		if (rc == 0)  {
+			_OutputStream.writeCharacters("0SECTION		Native configuration information for z/TPF\n");
+			_OutputStream.writeCharacters("NULL		==========================================\n");
+			_OutputStream.writeCharacters("1TPFHWMAXX	Highwater mark 64-bit heap 1MB Frames (limited by MAXXMMES): ");
+			_OutputStream.writeInteger(ztpfNativeStats.totalMAXXMMESRegion1MBFrames, "%u\n");
+			_OutputStream.writeCharacters("1TPFLIMMAXX	MAXXMMES keypoint A setting: ");
+			_OutputStream.writeInteger(ecbp2()->ce2proc->iproc_maxheap_64, "%u\n");
+			_OutputStream.writeCharacters("1TPFHWMMAP 	Highwater mark 64-bit MMAP heap 1MB Frames (limited by MAXMMAP): ");
+			_OutputStream.writeInteger(ztpfNativeStats.totalMMAPRegion1MBFrames, "%u\n");
+			_OutputStream.writeCharacters("1TPFLIMMMAP	MAXMMAP process setting (either keypoint A MAXMMAP or com.ibm.tpf.maxmmap property):       ");
```

Comment:
Please either remove the extra spaces here after the colon, or add sufficient spaces on all new lines so the values line up.

Reply:
Wasn't quite sure what you meant by this, but I made all the spaces consistent.